docs/cmake-project-view.md

```diff
@@ -0,0 +1,124 @@
+        "buildPreset": { 
+            "statusBarVisibility": "visible", "icon", "compact", "hidden", 
+            "statusBarLength": 20 
+        }, 
+        "buildTarget": { 
+            "statusBarVisibility": "visible", "compact", "hidden", 
+            "statusBarLength": 20 
+        }, 
+        "ctest": { 
+            "statusBarVisibility": "visible", "icon", "compact", "hidden", 
+            "statusBarLength": 20, 
+            "color": true, false, 
+            "projectStatusVisibility": "visible", "hidden" 
+        }, 
+        "testPreset": { 
+            "statusBarVisibility": "visible", "icon", "compact", "hidden", 
+            "statusBarLength": 20
+        },
+        "launchTarget": { 
+            "statusBarVisibility": "visible", "compact", "hidden", 
+            "statusBarLength": 20 
+        }, 
+        "debug": { 
+            "statusBarVisibility": "visible", "hidden", 
+            "projectStatusVisibility": "visible", "hidden" 
+        },
+        "launch": {
+            "statusBarVisibility": "visible", "hidden", 
+            "projectStatusVisibility": "visible", "hidden" 
+        }
+}
+```
+
+* `folder` is the active folder in your workspace. This is where the project is scoped to.
+* `configure` is associated with the CMake Configure action. It can only be found in the Project Status View and can't be seen in the status bar, hence no `statusBarVisibility` property value.
+* `configurePreset` is associated with the CMake Configure Preset. When selected, the user can modify their active CMake Configure Preset from the list detected in their CMakePresets.json and CMakeUserPresets.json (if found in project).
+* `kit` is associated with the active kit selected (when CMake Presets aren't present). When selected, the user can modify their active kit. To learn more about kits, please see [our kit documentation](https://github.com/microsoft/vscode-cmake-tools/docs/kits.md).
+* `variant` is associated with the active variant (when CMake Presets aren't present). When selected, the user can modify their active variant. Variant status does not show on the Project Status View, but will show in the status bar when set to `visible`. To learn more about variants, please see [our variant documentation](https://github.com/microsoft/vscode-cmake-tools/docs/variants.md).
+* `build` is associated with the CMake Build action. It invokes a CMake build on your build target using your build preset or variant.
+* `buildPreset` is associated with your active CMake Build Preset. When selected, the user can modify their active CMake Build Preset from the list detected in their CMakePresets.json and CMakeUserPresets.json (if found in project).
+* `buildTarget` is associated with your active CMake Build Target. When selected, the user can specify their active Build Target. This will be the target invoked when the user presses the `Build` icon in the status bar (if not hidden) or runs `CMake: Build Target` from the Command Palette.
```

Comment:
```suggestion
* `buildTarget` is associated with your active CMake Build Target. When selected, the user can specify their active Build Target. This will be the target invoked when the user presses the `Build` icon in the status bar (if not hidden) or runs `Cmake: Build Target` from the Command Palette.
```

Reply:
This should be `CMake`, with a capital M. So I don't think this suggestion should be taken.

Reply:
yeah, its still a capital M in the command palette, just not in the settings